objc/core/A0LockReact.m

```diff
@@ -159,17 +182,23 @@ - (void)authenticateWithConnectionName:(NSString *)connectionName options:(NSDic
     [authenticator authenticateWithConnectionName:connectionName parameters:parameters success:success failure:failure];
 }
 
+- (NSDictionary *)scopeParamterFromOptions:(NSDictionary *)options {
+    NSMutableDictionary *params = [[NSMutableDictionary alloc] init];
+    params[A0ParameterScope] = options[@"scope"];
```

Comment:
Why we should create a dict if we are only interested in the scope? Let's return the scope directly